src/app/nft-showcase/nft-showcase.component.html

```diff
@@ -17,7 +17,7 @@
   </div>
 </div>
 
-<div *ngIf="!loading && !nftCollections || !nftCollections.length" class="d-flex align-items-center justify-content-center">
+<div *ngIf="showcaseComingSoon()" class="d-flex align-items-center justify-content-center">
```

Comment:
we should avoid using functions for conditionals. ChangeDetection is faster for variables than it is for functions.

Reply:
```suggestion
<div *ngIf="!loading && !nftCollections?.length" class="d-flex align-items-center justify-content-center">
```

with the ?, we have safe access so `!(null)?.length` is truthy  and so is `!([])?.length`